### DISCUSSION TITLE
compile for armv6

### DISCUSSION THREAD

**Discussion by c-harris** [dotnet/runtime#64490] [Q&A]
I have been trying for 2 days to get this to compile for armv6. i can see the pull requests showing support has been added but i can't work out how to make use of it.

can anyone fill me in?

**Comment by huoyaoyuan** [comment]
ARMv6 support is in the progress to be added. See #63310 and its related issues.

**Reply by c-harris** [reply]
hey @huoyaoyuan  thanks for letting me know. the two referenced pr's in that issue have both been merged and i notice that armv6 has been added. have a jumped the gun here? is there still work to be done to finish adding support? 
`    public enum Architecture
    {
        X86 = 0,
        X64 = 1,
        Arm = 2,
        Arm64 = 3,
        Wasm = 4,
        S390x = 5,
        LoongArch64 = 6,
        Armv6 = 7,
    }
`

**Reply by huoyaoyuan** [reply]
I don't know detail about it. The necessary runtime code may be merged, but the related toolchain may not. You should really figure out it or ask someone knows it.
